github/issues_test.go

```diff
@@ -275,3 +275,14 @@ func TestIssuesService_Unlock(t *testing.T) {
 		t.Errorf("Issues.Unlock returned error: %v", err)
 	}
 }
+
+func TestIsPullRequest(t *testing.T) {
+	i := new(Issue)
+	if i.IsPullRequest() == true {
```

Comment:
Just curious about thoughts on removing `== true` (and similar `== false` below, but also negating it)? I often refer to tests as an example of how to use a function, and `if i.IsPullRequest() { ... }` if probably preferable than `if i.IsPullRequest() == true { ... }`?

I know both you and @shurcooL who already reviewed this knows you're doing, so curious why this wasn't flagged?